### DISCUSSION TITLE
Summary report on issues in cycle v4.20

### DISCUSSION THREAD

**Discussion by yeneastgate** [onOffice-Web-Org/oo-wp-plugin#725] [General]
@fredericalpers 
Currently, I see that the number of issues in cycle v4.20 is still quite large (I need another 7.5 working days to work on it).
Here's a report on the status of these issues:

![image](https://github.com/onOffice-Web-Org/oo-wp-plugin/assets/104892108/60cb1d4f-7917-43ed-846e-766236c7510d)

So, we will prioritize the completion of v4.20 first, correct?
After that, we will focus on investigating v4.21, right?
Please let me know your opinion about this, thanks!

**Comment by fredericalpers** [comment]
@yeneastgate yes, please prioritize v4.20 first :) I will get back to you on the issues asap. Thank you

**Reply by yeneastgate** [reply]
@fredericalpers I got it, thanks!
